aptos-move/framework/src/natives/aggregator_natives/aggregator_v2.rs

```diff
@@ -129,16 +151,30 @@ fn native_try_add(
     mut args: VecDeque<Value>,
 ) -> SafeNativeResult<SmallVec<[Value; 1]>> {
     debug_assert_eq!(args.len(), 2);
-
     context.charge(AGGREGATOR_V2_TRY_ADD_BASE)?;
-    let aggregator_context = context.extensions().get::<NativeAggregatorContext>();
-    let mut aggregator_data = aggregator_context.aggregator_data.borrow_mut();
-    let value = pop_value_by_type(&ty_args[0], &mut args)?;
-    let (id, limit) = pop_aggregator_fields_by_type(&ty_args[0], &mut args)?;
-    let aggregator = aggregator_data.get_aggregator(AggregatorVersionedID::V2(id), limit)?;
-    Ok(smallvec![Value::bool(
-        aggregator.try_add(aggregator_context.resolver, value)?
-    )])
+
```

Comment:
There is no if-else statement checking aggregator_api_enabled flag.

Reply:
there is no need - we are gating constructors. 

when flag is disabled, nobody can create aggregators/snapshots, and so nobody can call this functions

after we enable the flag, and people create aggregators/snapshots, we can disable new construction, but shouldn't break contracts that created those. for that we have aggregator_execution_enabled flag which disables all complexities and should always work.